src/components/AppHeader.tsx

```diff
@@ -10,8 +10,16 @@ import {
 } from "grommet";
 import { Down } from "grommet-icons";
 import { UserMenu } from ".";
+import { User, Items } from ".";
 
-export const AppHeader = ({ appName, appIcon, userSession, open }) => (
+interface AppHeaderProps {
```

Comment:
This is strange interface definition. Please consider strong types (simmilar to what would be generated using graphqlcodegenerator)